build.gradle

```diff
@@ -113,7 +113,7 @@ ext.tempto_runner = project(':tempto-runner')
 ext.tempto_ldap = project(':tempto-ldap')
 ext.tempto_kafka = project(':tempto-kafka')
 ext.expected_result_generator = project(':expected-result-generator')
-ext.tempto_version = '1.55-SNAPSHOT'
+ext.tempto_version = '1.54-SNAPSHOT'
```

Comment:
Are we intending to roll back the commits on master to fix this? Or just attempt the action again & re-tagging? We probably need to delete the git tag before retrying

Reply:
We should republish 1.54 since it was not released past staging. I thought re-tagging would occur with this as well